clients/client-py/README.md

```diff
@@ -9,45 +9,59 @@ This library is a complete interface to Taskcluster in Python.  It provides
 both synchronous and asynchronous interfaces for all Taskcluster API methods,
 in both Python-2 and Python-3 variants.
 
-## Usage
+## Authenticated Client Example
 
-For a general guide to using Taskcluster clients, see [Calling Taskcluster APIs](https://docs.taskcluster.net/docs/manual/using/api).
+For a general guide to using Taskcluster clients, see [Calling Taskcluster APIs](https://docs.taskcluster.net/docs/manual/using/api).     
+                                                                      
+#### Acquire Access Token for Client
 
-### Setup
+The [Community Taskcluster instance]( https://community-tc.services.mozilla.com) will be used as an example. You can use your github account to login. Start the process here: [Create client](https://community-tc.services.mozilla.com/auth/clients/create?scope=)
 
-Before calling an API end-point, you'll need to create a client instance.
-There is a class for each service, e.g., `Queue` and `Auth`.  Each takes the
-same options, described below.  Note that only `rootUrl` is
-required, and it's unusual to configure any other options aside from
-`credentials`.
+Fill in the parameters:
+
+* **Client ID** - whatever you like, but must be prefixed with your user id. Example: `github/1234567|username/clientname`
```

Comment:
Good question.  I had assumed they could login to a taskcluster instance; without that assumption the docs are not much help.

Reply:
Ah, sorry, what you called "user id" is a clientId.  So how would they know what their equivalent of `github/12334567|username` is?  And would they be confused if it didn't start with `github`?  Will they guess to replace `username` with their github username, not realizing that they need to replace `1234567` with their github userid, which is not easily discovered?

The larger point is, our experience with docs that are specific to particular circumstances is that while they are beneficial for people matching those circumstances, they are detrimental to people not matching those circumstances, and the overall support burden on us actually increases.

Reply:
> Will they guess to replace username with their github username, not realizing that they need to replace 1234567 with their github userid, which is not easily discovered?

I do not know.  The Community task cluster names are quite complicated; they look like a mini language operators rather than names.  It took a while for me to realize that `assume:` is not just another prefix; it had meaning. There are also confusing rules that force client id to have user id prefix. Sorry I will not be of help.